packages/wallet-connectors/src/index.tsx

```diff
@@ -76,7 +77,13 @@ export const avalancheCore = new AvalancheCoreConnector({
   supportedChainIds: SUPPORTED_EVM_CHAINS_ID,
 });
 
-export { NearConnector, HashConnector, HashConnectEvents, hashconnectEvent, WalletConnectConnector, NetworkConnector };
+export const freighter = new FreighterConnector({
+  supportedChainIds: [7566437, 7566438],
+  normalizeAccount: false,
+  normalizeChainId: false,
+});
```

Comment:
this is already defined in `src/FreighterConnector/index.ts` , what is purpose to define it again ?